ArduCopter/mode_althold.cpp

```diff
@@ -40,7 +40,7 @@ void ModeAltHold::run()
 
     // get pilot desired climb rate
     float target_climb_rate = get_pilot_desired_climb_rate();
-    target_climb_rate = constrain_float(target_climb_rate, -get_pilot_speed_dn(), g.pilot_speed_up);
+    target_climb_rate = constrain_float(target_climb_rate, -get_pilot_speed_dn(), constrain_int16(g.pilot_speed_up, 50, 500));
```

Comment:
Wait, this doesn't look right?

Why are we constraining pilot climb rate between 0.5m/s and 5m/s here?  Some vehicles and scenarios could be *well* outside these ranges!